app/views/decidim/newsletters/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<% @cell = cell(
```

Comment:
Hey @paulinebessoles,

Comparing this file with [the original](https://github.com/decidim/decidim/blob/release/0.24-stable/decidim-core/app/views/decidim/newsletters/show.html.erb) I don't see significative changes, does this monkey patch is still needed ?

Reply:
You're right, I delete it!